### ISSUE TITLE
Single-line if statements with parentheses won't be formatted

### ISSUE THREAD

**Issue by FooIbar** [pinterest/ktlint#3064]
```kotlin
fun foo() {
    if (true)(0)
    if (true)(0)else 1
}
```

## Expected Behavior
```kotlin
fun foo() {
    if (true) (0)
    if (true) (0) else 1
}
```

## Observed Behavior
Not formatted

## Your Environment
* Version of ktlint used: 1.7.0

**Comment by paul-dingemans** [comment]
Tnx for reporting. This is not a regression bug of the `1.7.0` release. I cannot find an older ktlint release in which the properly worked. Please let me know, if you can reproduce with older version.

**Comment by paul-dingemans** [comment]
The problem is not related to the parenthesis. Examples below is also not reformatted:
```
    if (true)false
    if (true)true else(1)
```
The problem is that the `then` should be surrounded by either a space, or a newline.
